internal/transport/grpchttp2/errors.go

```diff
@@ -30,6 +30,7 @@ import "fmt"
 // [GOAWAY]: https://httpwg.org/specs/rfc7540.html#GOAWAY
 type ErrCode uint32
 
+// Error Codes defined by the HTTP/2 Spec.
```

Comment:
I'm not entirely sure how the linter works, but do you know if it expects you to add a docstring for each individual exported variable?